### DISCUSSION TITLE
jobs.list_runs does not return 'latest_repair_id'

### DISCUSSION THREAD

**Discussion by btglau** [databricks/databricks-sdk-py#158] [Q&A]
Hello,

I wrote a script that can repair all the runs for a given job_id. However, when I have to repair for a second time, I come across this error:

`w.jobs.repair_run(j.run_id, rerun_all_failed_tasks=True)`
`databricks.sdk.core.DatabricksError: The latest repair ID needs to be provided in order to create a new repair`

I cannot find any output in jobs object that has this information. Can you provide some insight here? Thanks!

**Comment by KoningJasper** [comment]
@btglau You need to pass the latest repair_id which is returned when you created it, or can be retrieved in the repair_history field using get_run on the job run, see https://docs.databricks.com/api/azure/workspace/jobs/repairrun and https://docs.databricks.com/api/azure/workspace/jobs/getrun.

**Comment by btglau** [comment]
`repair_history` is the one, thanks. When I run my script to repair runs, they may fail later, so I don't have access to the job object. Would you say it is best practice to save these runs in a file? I'd rather get all the information I need from calls to the SDK.

**Comment by saipuneeth25** [comment]
I couldn't find the repair history from even though i have used get run API by using include_history to 'True', Could you please help me how to check or debug with screenshots
